fili-core/src/test/java/com/yahoo/bard/webservice/application/TestBinderFactory.java

```diff
@@ -172,7 +176,13 @@ public ConfigurationLoader getConfigurationLoader() {
      * @return the datasource metadata service
      */
     public DataSourceMetadataService getDataSourceMetaDataService() {
-        return buildDataSourceMetadataService();
+        return new TestDataSourceMetadataService();
+    }
+
+    @Override
+    protected DataSourceMetadataService getDataSourceMetadataService() {
+        this.dataSourceMetadataService = new TestDataSourceMetadataService();
```

Comment:
We should only have 1 of these methods, not both (likely the latter one, not the former)